configure.ac

```diff
@@ -358,7 +358,7 @@
 
     #Enable support for gcc compile time security options. There is no great way to do detection of valid cflags that I have found
     #AX_CFLAGS_GCC_OPTION don't seem to do a better job than the code below and are a pain because of extra m4 files etc.
-    #These flags seem to be supported on CentOS 5+, Ubuntu 8.04+, and FedoreCore 11+
+    #These flags seem to be supported on CentOS 5+, Ubuntu 8.04+, and FedoraCore 11+
```

Comment:
It probably does, although it might be best not to do that in this PR. I'm happy to drop the change and have a distinct PR for it.

Anytime a PR like this does more than simply "fix spelling", it makes it harder for everyone (both reviewers now as well as anyone looking later and wondering what happened).